android/app/src/main/java/org/fossasia/openevent/utils/CompressImage.java

```diff
@@ -0,0 +1,30 @@
+package org.fossasia.openevent.utils;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Created by free4murad on 3/9/17.
+ */
+
+public class CompressImage {
+    public static String compressImage(String file) {
+        URL url;
+        String temp = "";
+        int indexOfPath = -1;
+        try {
+            url = new URL(file);
+        } catch (MalformedURLException e) {
+            return file;
+        }
+        indexOfPath = file.indexOf(url.getPath());
+        if(!url.getHost().endsWith(".rsz.io")) {
+            temp = file.substring(0, indexOfPath) + ".rsz.io" + file.substring(indexOfPath) + "?width=300";
```

Comment:
Sorry for the late reply just saw this.
Actually we have to keep in mind the large devices like tabs.
Also the size difference between images with width 200 and 300 is very less.